README.md

```diff
@@ -32,7 +32,7 @@ It is also useful to enable `loc` generation for dev environment, to see for exa
 
 ```json
 {
-  "plugins": [["effector/babel-plugin", { "addLoc": true }]]
+  "plugins": ["effector/babel-plugin", { "addLoc": true }]
```

Comment:
But this is not a typo 🤔 

This is how options are provided to babel plugins
https://github.com/effector/next#dev-tools-integration